vm/onboarding.go

```diff
@@ -8,12 +8,23 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"strconv"
+	"strings"
 	"time"
 
 	tp "github.com/kubearmor/KVMService/src/types"
 	"sigs.k8s.io/yaml"
 )
 
+type KVMSEndpoint struct {
```

Comment:
@wazir-ahmed  Kindly fix this linter warning. Add comment for this func

Reply:
Removed the `struct` definition from here and directly used the struct from `github.com/kubearmor/kvm-service/src/types` package.